config/crawler.yml.example

```diff
@@ -54,6 +54,10 @@
 ## The maximum depth that Crawler will follow links to.
 #max_crawl_depth: 2
 #
+## Scheduling using cron expressions
+#schedule:
+#  - interval: "0 12 * * *"     # every day at noon
```

Comment:
```suggestion
#  - cron: "0 12 * * *"     # every day at noon
```

Imo `interval` references fixed-time gaps and doesn't really fit to cron expression

Reply:
I want to avoid calling it `cron` because the lib I'm using also supports other scheduling types. So in the future I want to be able to have things like this:

```yaml
# cron schedule
schedule:
    type: cron
    interval: "* * * * *"

# intervals
schedule:
    type: interval
    interval: "30m"
```

But I agree seeing your review that `interval` isn't appropriate. Maybe `pattern`?

Reply:
I've changed it to `pattern` for now, let me know if it still reads weird

Reply:
it feels better than interval, and it won't be confusing if we start supporting `interval` type. there is also option calling it `expression` - up to you :)

Reply:
I'm okay with `expression` as well, but is `30m` (30 minutes) an expression? 🤔

Reply:
(I actually don't know haha)

Reply:
Sticking with `pattern` for now 👍🏻